### DISCUSSION TITLE
How to build a controller with GitHub Copilot

### DISCUSSION THREAD

**Discussion by tuves** [community/community#66377] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

Building a controller function in Node.js can be time-consuming and tedious. :hot_face: But did you know you can now use GitHub Copilot to help you create your controllers—faster and better than ever? :muscle: Our September edition of _The GitHub Insider_ newsletter will walk you through how to do this in four easy steps! Don't miss it, subscribe now. :newspaper:

[Subscribe to the newsletter](https://resources.github.com/newsletter/)

**Comment by reeveng** [comment]
Did you know that reverting the new homepage could restore the peace and order of our universe! :fire: :shipit:

**Reply by reeveng** [reply]
Why go back to the old GitHub homepage when you can have GitHub Copilot build a time machine for you to restore peace, pls?
